modules/api_reference/devportal_api_reference.module

```diff
@@ -22,36 +22,35 @@ use Symfony\Component\HttpFoundation\Request;
  *   An array of node bundles.
  */
 function devportal_api_reference_bundles() {
-  return ['api_reference'];
+  define('devportal_api_reference_bundles', ['api_reference']);
```

Comment:
Why is this workaround is required with constants? :roll_eyes:  For me, it seems odd defining a constant within a function which who knows when gets called...

Reply:
I don't know the reason, this was in the task: refactor devportal_api_referenced_bundles() to const array. It is called in the next function: devportal_api_reference_menu_links_discovered_alter() and in the APIRefBundleAccessCheck class. So should I eliminate this devportal_api_reference_bundles() function or just leave it as it was originally?

Reply:
What about replacing this whole `devportal_api_reference_bundles()` function with a constant, and use that constant whereever this function was called?

Reply:
Please also check [Drupal CS regarding constants](https://www.drupal.org/docs/develop/standards/coding-standards#naming).